v3/src/components/graph/adornments/movable-value/movable-value-adornment-registration.tsx

```diff
@@ -35,25 +36,26 @@ const Controls = () => {
       kMovableValueUndoRemoveKey, kMovableValueRedoRemoveKey
     )
   }
-
+  /**
+   * Todo: The two buttons are problematic wrt style.
+   * 1. The className is not being applied to the buttons.
+   * 2. The font-weight defined in buttonStyle is not being applied to the buttons.
+   * 3. There should be a space between the two buttons.
+   * Note that moving this code to the GraphMeasureGroup component doesn't solve the problem.
+   */
```

Comment:
Does this comment need to be updated since you fixed that style selector issue? I think at least #1 is fixed, and that note at the end doesn't matter anymore?